Source/Dafny/Verifier/Translator.TrStatement.cs

```diff
@@ -1495,13 +1497,7 @@ public partial class Translator {
           if (includeTerminationCheck) {
             AddComment(loopBodyBuilder, s, "loop termination check");
             Bpl.Expr decrCheck = DecreasesCheck(toks, types, types, decrs, oldBfs, loopBodyBuilder, " at end of loop iteration", false, false);
-            string msg;
-            if (s.InferredDecreases) {
-              msg = "cannot prove termination; try supplying a decreases clause for the loop";
-            } else {
-              msg = "decreases expression might not decrease";
```

Comment:
This used to say "expression", but the new code says "clause"

Reply:
Yeah, true. Which do you prefer? The previous code sometimes used one and sometimes the other.